examples/mongo.yaml

```diff
@@ -11,7 +11,8 @@ spec:
   backups:
     gcs:
       bucket: "mongo-backups"
-      serviceaccount:
+      service_account:
+        name: MONGO_SERVICE_ACCOUNT
```

Comment:
Why does the service account need a name here? Note this is a Google Cloud service account (JSON key file), not a Kubernetes service account.

Reply:
@ChrisTerBeke the API model that we use (`V1EnvVar` or so) has a required name